src/oidcop/oauth2/add_on/dpop.py

```diff
@@ -58,6 +59,7 @@ def create_header(self) -> str:
         payload = {k: self[k] for k in self.body_params}
         _jws = JWS(payload, alg=self["alg"])
         _headers = {k: self[k] for k in self.header_params}
+        self.key.kid = ""
```

Comment:
Why is this needed?

Reply:
actually it should be rfc7836 aware
https://github.com/IdentityPython/JWTConnect-Python-CryptoJWT/blob/815d5b6a3d80852fcaaff5f8227f735124e06f2c/src/cryptojwt/jwk/__init__.py#L274

Reply:
Because you don't want it go looking for a key with a specific `kid`.
This since the key you get in the headers doesn't have a `kid`.